test/image/compare_pixels_test.js

```diff
@@ -104,6 +104,7 @@ var FLAKY_LIST = [
     'treemap_coffee',
     'treemap_textposition',
     'treemap_with-without_values_template',
+    'treemap_with-without_values',
```

Comment:
Despite black listing this mock, the image test still fails!

Reply:
Do you have any idea why?

Reply:
I am trying to fix it.

Reply:
Would it be ok if I simply update the baseline?

Reply:
You could pick https://github.com/plotly/plotly.js/commit/2589451c1c0734de9c0268b7af6093bd6f94d461.

Reply:
Wanna push that commit on this branch? It's your work afterall. Thanks very much for looking into this!

Reply:
Not a big deal. 
OK. I'll push.